embark-ui/public/index.html

```diff
@@ -20,7 +20,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Embark UI</title>
+    <title>Embark Cockpit</title>
```

Comment:
I think this should be just "Cockpit" then no?

Reply:
Both are fine IMO